core-build/gulp-core-build/src/tasks/JestReporter.ts

```diff
@@ -6,15 +11,50 @@ const DEFAULT_REPORTER: any = require('jest-cli/build/reporters/default_reporter
  * rush does not get confused.
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-class JestReporter extends (DEFAULT_REPORTER as { new (globalConfig: any): any }) {
+class JestReporter extends (DEFAULT_REPORTER as { new (globalConfig: Jest.GlobalConfig): any }) {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public constructor(globalConfig: any) {
+  private _globalConfig: Jest.GlobalConfig;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public constructor(globalConfig: Jest.GlobalConfig) {
     super(globalConfig);
+    this._globalConfig = globalConfig;
+
+    this._err = this._out;
```

Comment:
I assume `_out` comes from the base class? Is it always an error?

Can you put a comment here? This reads as a little odd.

Reply:
so _out is what they pipe normal output to in the default_reporter, and _err is the stderr. I'm basically trying to prevent the reporter from writing to the error output (similar to how the log() method is overriden to prevent it in other scenarios.

Looking into it, the "useStderr" var is false by default in the globalconfig, so this should be a non-issue anyway. Removed